en/intro_to_command_line/README.md

```diff
@@ -72,7 +72,7 @@ Each command will be prepended by this sign and one space, but you don't have to
 
 The part up to and including the `$` or the `>` is called the *command line prompt*, or *prompt* for short. It prompts you to input something there.
 
-In the tutorial, when we want you to type in a command, we will include the `$` or `>`, and occasionally more to the left. You can ignore the left part and type in the command which starts after the prompt.
+In the tutorial, when we want you to type in a command, we will include the `$` or `>`, and occasionally more to the left. Ignore the left part and only type in the command, which starts after the prompt.
```

Comment:
:+1:

Reply:
This is a very common mistake and many students copy or type the entire line. I think this version is better than what we used to have but I wonder if there's even a better way. Perhaps it's a separate issue though.

Reply:
> Perhaps it's a separate issue though.

Indeed. Let the perfect not be the enemy of the good. The change suggested here is already a good improvement. So unless we can come up with something better right now, let's merge this and handle further improvement in a separate issue/PR.